projects/packages/connection/legacy/class-jetpack-options.php

```diff
@@ -125,6 +125,7 @@ public static function get_option_names( $type = 'compact' ) {
 			'identity_crisis_url_secret',          // (array) The IDC URL secret and its expiration date.
 			'identity_crisis_ip_requester',        // (array) The IDC IP address and its expiration date.
 			'dismissed_welcome_banner',            // (bool) Determines if the welcome banner has been dismissed or not.
+			'dismissed_welcome_tour',              // (bool) Determines if the welcome tour has been dismissed or not.
```

Comment:
Since now we have two welcome elements that uses similar backend code, I think we could use a more clarifying comment, just to best identify the difference between the banner and the tour